languages/rust/exercises/concept/options/tests/options.rs

```diff
@@ -0,0 +1,89 @@
+use options::*;
+
+#[test]
+fn test_reviving_dead_player() {
+    let level: u32 = 34;
+    let dead_player = Player {
+        health: 0,
+        mana: Some(0),
+        level,
```

Comment:
```suggestion
        34,
```

Just use a constant here; you don't need to break out a separate `level` variable.